gitpractice/youtubetask/DOM.js

```diff
@@ -0,0 +1,85 @@
+var url = "https://www.googleapis.com/youtube/v3/search?key=AIzaSyBlF18tOPAMKBDIgLMRECjH7le_YJ_En4M&type=video&part=snippet&maxResults=15&q="
+var searchBox = document.querySelector('#Search-box')
+var searchBtn = document.querySelector('#searchBtn')
+var nextBtn = document.querySelector("#next-btn");
+var previousBtn = document.querySelector("#previous-btn");
+let page = document.querySelector("#page-content");
+let counter = 0;
+let array;
+let limit = 4;
+let start = 0;
+var c = 0;
+searchBtn.addEventListener('click', event => {
+    var keyword = searchBox.value;
+    if (keyword == 0) {
+        alert("please type something")
+        return
+    }
+    page.innerHTML = "";
+    fetch(url + keyword)
+        .then(response => {
+            return response.json()
+        }).then(result => {
+            counter = 0;
+            start = 0;
+            c++;
+            console.log(result);
+            data = result.items;
+            showDetails(data);
+        })
+})
+let showDetails = function(data) {
+    displayDetails(data);
+}
+
+function displayDetails(data) {
+    if (start + counter >= 15)
+        return;
+
+    page.innerHTML = "";
+    start += counter;
+    counter = 0;
+    for (let i = start; i < start + limit && i < 15; i++) {
```

Comment:
use meaningfull names use iterator instead of i

Reply:
yes i have changed it